Documentation/CodeSnippets/ExtensionConfiguration/TcaOverridePages.rst.txt

```diff
@@ -6,19 +6,6 @@
 
    <?php
    defined('TYPO3') || die();
-   
+
```

Comment:
the file is now functionally empty and can be removed.

Reply:
done